test/cmds/serve/serve.test.js

```diff
@@ -0,0 +1,83 @@
+const EventEmitter = require('events');
+const http = require('http');
+const serve = require('../../../dist/src/cmds/serve');
+const serveUtils = require('../../../dist/src/cmds/serve/serveUtils');
+const fsUtils = require('../../../dist/src/utils/validate-fs');
+
+const mockArgv = {
+  root: '.',
+  port: 8081,
+};
+
+describe('serve', () => {
+  describe('Starts a local, static HTTP server on the given port with the given root directory.', () => {
+
+    let serverEmitter;
```

Comment:
I should've thought about this earlier, but `mockServer` is an even better name for this variable. That it's an emitter is mostly beside the point.